### PR TITLE
Disable failing assert in failing test DebuggerTests.Threads Assert.AreEqual (ThreadState.Stopped, e.Thread.ThreadState)

### PR DESCRIPTION
**Pull request by jaykrell**
https://github.com/mono/mono/issues/11416

### CODE REVIEW

**Comment by vargaz** [comment]
That test used to run reliably, what changed ?

**Comment by jaykrell** [comment]
I don't know. Seems to be just Mac/x86. Disable it only there?

**Comment by jaykrell** [comment]
@monojenkins build failed

File: mcs/class/Mono.Debugger.Soft/Test/dtest.cs [hunk 1]
```diff
@@ -3225,7 +3225,8 @@ Event step_out_await (string method, Event e)
 
 		e = GetNextEvent ();
 		Assert.IsInstanceOfType (typeof (ThreadDeathEvent), e);
-		Assert.AreEqual (ThreadState.Stopped, e.Thread.ThreadState);
+//https://github.com/mono/mono/issues/11416
```
Comment (luhenry) [inline mcs/class/Mono.Debugger.Soft/Test/dtest.cs]: Fix indent.

**Comment by lewurm** [comment]
@monojenkins build failed

**Comment by lewurm** [comment]
@monojenkins squash

**Comment by jaykrell** [comment]
@monojenkins build failed

**Comment by jaykrell** [comment]
@monojenkins build  OS X x64 C++

**Comment by jaykrell** [comment]
@monojenkins build failed

**Comment by jaykrell** [comment]
@monojenkins build failed

**Comment by jaykrell** [comment]
@monojenkins build OS X x64 C++

**Pull request merged**